backend/elasticsearch/src/main/java/org/hibernate/search/backend/elasticsearch/index/impl/ElasticsearchIndexManagerImpl.java

```diff
@@ -222,6 +230,30 @@ public ElasticsearchIndexDescriptor descriptor() {
 		return model;
 	}
 
+	@Override
+	public List<? extends AnalysisToken> analyze(String analyzerName, String terms) {
+		return Futures.unwrappedExceptionJoin(
+				analyzeAsync( analyzerName, terms, OperationSubmitter.rejecting() ).toCompletableFuture() );
+	}
+
+	@Override
+	public AnalysisToken normalize(String normalizerName, String terms) {
+		return Futures.unwrappedExceptionJoin(
+				normalizeAsync( normalizerName, terms, OperationSubmitter.rejecting() ).toCompletableFuture() );
```

Comment:
If you're going to block anyway in the end, you might as well use `OperationSubmitter.blocking()`.

Reply:
true 👍🏻 I was just following the example for `computeSizeInBytesAsync` 😃

Reply:
`computeSizeInBytes` you mean? That one probably should be fixed as well... but it's not that important